pkg/cmd/clustertask/start_test.go

```diff
@@ -513,21 +687,6 @@ func Test_ClusterTask_Start(t *testing.T) {
 			wantError:   false,
 			goldenFile:  true,
 		},
-		{
-			name: "Dry run with no output v1beta1",
-			command: []string{"start", "clustertask-2",
-				"-i", "my-repo=git",
-				"-o", "code-image=output-image",
-				"-l", "key=value",
-				"-s=svc1",
-				"--dry-run",
-			},
-			dynamic:     seeds[4].dynamicClient,
-			input:       seeds[4].pipelineClient,
-			inputStream: nil,
-			wantError:   false,
-			goldenFile:  true,
-		},
```

Comment:
Not sure what this test was intended for. The name suggests it's a `v1beta1` test, but it's part of the `v1alpha1` suite.